ansible_dynamic_inventory/cli.py

```diff
@@ -15,7 +15,10 @@ def _parse_program_args():
     parser.add_argument('--list', action='store_true', help='print dynamic inventory')
     parser.add_argument('--plantuml', action='store_true', help='print plantuml format')
     parser.add_argument('--config', action='store', help='path to ansible_dynamic_inventory.ini')
-    return vars(parser.parse_args())
+    args = vars(parser.parse_args())
+    if args['config'] == None and 'ANSIBLE_DYNAMIC_INVENTORY_INI_PATH' in os.environ:
```

Comment:
__[flake8]__
* [E711] comparison to None should be 'if cond is None:'

<!-- Comment by [SideCI](https://sideci.com) -->